### PR TITLE
Automate our release process

### PR DESCRIPTION
**Pull request by lovedeep92**
**JIRA:**
See ticket here - https://jira.rax.io/browse/MNRVA-798

**Description:**

1) Created new script.js file.
2) NPM pack the project and rename the tgz file to the name of the package.
3) Create release based on zip and also uploading tgz file to release using putasset library used for adding assets to release.
4) Git ignore this zip so it is not checked in only created locally.

## Screenshots:
(if applicable)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,13 @@
     "visual-test": "node backstop.js --test --singleRun",
     "pretest": "npm run build",
     "serve": "node -e \"require('./bin/server.js').start()\"",
-    "reference": "node backstop.js --inspect-brk --reference --singleRun"
+    "reference": "node backstop.js --inspect-brk --reference --singleRun",
+    "pack": "(exec npm pack) && (mv \"$(npm pack | tail -n 1)\" $npm_package_name.tgz)",
+    "release": "./script.js"
```
Comment (FarajiA) [inline package.json:18]: Can we make _npm run release_  the only step required for a release.

**Review by FarajiA — APPROVED** [review]
Final step would be updating the release instructions in the README.

File: package.json (hunk 1 above)
Comment (lovedeep92) [reply package.json]: DONE !!

**Comment by lovedeep92** [comment]
Yup. Updated the README.md file as well !!

**Pull request merged**